ups/analysis_tools.table

```diff
@@ -3,6 +3,7 @@
 # - Common third-party packages can be assumed to be recursively included by
 #   the "base" package.
 setupRequired(base)
+setupRequired(pipe_tasks)
```

Comment:
Since you are keeping up with this anyway, can you add a required for pipe_base and daf_butler ?